python/tskit/tables.py

```diff
@@ -1088,6 +1088,9 @@ def __init__(self, max_rows_increment=0, ll_table=None):
             ll_table = _tskit.NodeTable(max_rows_increment=max_rows_increment)
         super().__init__(ll_table, NodeTableRow)
 
+    def _has_discrete_times(self):
```

Comment:
This is going to sound picky, but I'm not a fan of one-use functions like this. If we could see ourselves using this a lot then I think it's worth it, but I don't see us reusing these so it isn't worth breaking up the flow and adding to the large list of methods on a Table.

Reply:
I'm happy to go with whatever you suggest. I'm just following what @jeromekelleher suggested (that's me passing the buck)

Reply:
Sure, that would be good.

Reply:
Done. Much tidier now, good call.